R/fit.R

```diff
@@ -195,6 +196,15 @@ refit_multiple_optimizers <- function(fit,
 #' However, please be cautious because this can lead to convergence failure
 #' when using an unstructured covariance matrix and there are no observations
 #' at the missing visits.
+#' The `method` and `cov` arguments will decide the covariance matrix and degree of freedom calculations.
+#' If `method` is "Kenward-Roger" then only "Kenward-Roger" or "Kenward-Roger-Linear" are allowed for `cov`.
+#' The `cov` argument can be `NULL` to use the default covariance method depending on the `method`
+#' used for degree of freedom, see the following table:
```

Comment:
```suggestion
#' used for degrees of freedom, see the following table:
```
(basically there is not singular "degree" of freedom, because it is almost always more than 1.0 degree :-))